solutions/landing-zone/README.md

```diff
@@ -86,170 +86,185 @@ The following resources will be deployed.
-    ```
+ The section around billing that looks like the following and can either be commented out or deleted.
+  ```
     billingAccountRef:
         # Replace "${BILLING_ACCOUNT_ID?}" with the numeric ID for your billing account
         external: "${BILLING_ACCOUNT_ID?}" # kpt-set: ${billing-id}
-    ```
+  ```
 
-    This will cause the project to spin up with no attached billing id and any service that requires billing to be enabled will pause deployment until billing is enabled. Billing can be added by a user with Billing User permission in the Billing UI. If you do not remove this section the project will fail to create.
+ This will cause the project to spin up with no attached billing id and any service that requires billing to be enabled will pause deployment until billing is enabled. Billing can be added by a user with Billing User permission in the Billing UI. If you do not remove this section the project will fail to create.
 
-    You can view the status of any deployed object by running `kubectl get gcp` when connected to the Config Controller instance. If an object is pending or is displaying an error you can investigate by copying the name of the object and running the describe command.
+ You can view the status of any deployed object by running `kubectl get gcp` when connected to the Config Controller instance. If an object is pending or is displaying an error you can investigate by copying the name of the object and running the describe command.
 
-    For example `kubectl describe storagebucket.storage.cnrm.cloud.google.com/audit-sink-audit-prj-12345`.
+ For example `kubectl describe storagebucket.storage.cnrm.cloud.google.com/audit-sink-audit-prj-12345`.
 
-    ## GitOps 
-    ### Git
+ ## GitOps 
+ ### Git
 
-      Deploy Infrastructure via GitOps using Anthos Config Management
+   Deploy Infrastructure via GitOps using Anthos Config Management
 
-      To start you will need a git repo, for this guide we will be using Cloud Repositories but you could easily use Gitlab, or Github. The instructions have been modified from the config controller setup guide located [here](https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup#set_up_gitops)
+   To start you will need a git repo, for this guide we will be using Cloud Repositories but you could easily use Gitlab, or Github. The instructions have been modified from the config controller setup guide located in [Manage Google Cloud resources with Config Controller](https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup#manage-resources) and [Setup GitOps](https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup#set_up_gitops). 
+   
+   Skip this step if not using Cloud Repositories.
 
-      1. Create Source Control Service (skip this if not using Cloud Repositories)
-          ```
+ #### 1a. Create Source Control Service 
+   ```
```

Comment:
YAML examples are all over indented. Is there a reason for that?

Reply:
If the code sections are past 4 chars - they don't render when within sections - 4 is the max - had to reduce them

Reply:
I left the code sections as-is with the existing indentation whitespace - to avoid more merge conflicts
Only the triple quoting above/below needed to be moved
I would like and can remove all the whitespace from the original though - code runs OK with the whitespace for now
```
from

      1. Create Source Control Service (skip this if not using Cloud Repositories)
          ```
          # service.yaml

          apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
          
 to
 #### 1a. Create Source Control Service 
   ``
          # service.yaml

          apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
          
          
  ```

Reply:
ETA - merging in 68min